app/views/facilities/index.html.haml

```diff
@@ -21,3 +21,19 @@
 
 - else
   .alert.alert-info= t(".empty", items: Facility.model_name.human(count: 2))
+
+- if @products.any?
+  %h4= text(".recently_purchased")
+  %table.table.table-striped.table-hover.product_list
```

Comment:
I definitely get why you're using a table here; however, we don't typically use tables when listing products in this type of view, so it kind of breaks the style idiom for users, if that makes sense.

I would suggest modeling this after https://github.com/tablexi/nucore-open/blob/master/app/views/facilities/_product_list.html.haml#L7 -- that's what we use in `facilities/show`, so the link locations and style will match what the user expects.